### PR TITLE
Update android doc.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Reopen of #271

### CODE REVIEW

File: documentation/src/asciidoc/setting-up.adoc [hunk 1]
```diff
@@ -152,3 +152,82 @@ class CalculatorSpec: Spek({
...
+
+[source,groovy,subs="attributes"]
+.build.gradle
+----
+buildscript {
+    ...
+
+    // 1. Add the android-junit5 Gradle plugin as a classpath dependency
+    dependencies {
+        classpath "de.mannodermaus.gradle.plugins:android-junit5:${junitPlatformVersion}"
```
Comment (artem-zinnatullin) [inline documentation/src/asciidoc/setting-up.adoc:168]: Maybe explicitly add AGP 3.x?

**Review by artem-zinnatullin — COMMENTED** [review]
LGTM

File: documentation/src/asciidoc/setting-up.adoc (hunk 1 above)
Comment (arturdryomov) [reply documentation/src/asciidoc/setting-up.adoc:168]: Not everybody uses it, you know. And I don’t know what you meant by that.

File: documentation/src/asciidoc/setting-up.adoc [hunk 2]
```diff
@@ -152,3 +152,82 @@ class CalculatorSpec: Spek({
 IMPORTANT: As mentioned in the <<ide-support>> section, the IDEA plugin won't work if you're using the JUnit 4 runner.
 
 
+
+[[setting-up-android]]
+== Setting up Android
```
Comment (arturdryomov) [inline documentation/src/asciidoc/setting-up.adoc:157]: As a matter of fact it did not help us using AS 3.0 RC 1.

Also, specify the AS versions supported by this.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: documentation/src/asciidoc/setting-up.adoc (hunk 1 above)
Comment (mannodermaus) [reply documentation/src/asciidoc/setting-up.adoc:168]: On a different note: The android-junit5 plugin uses an independent version from the JUnit Platform, so it shouldn't re-use the `$junitPlatformVersion` variable.

File: documentation/src/asciidoc/setting-up.adoc (hunk 2 above)
Comment (raniejade) [reply documentation/src/asciidoc/setting-up.adoc:157]: Is it safe to assume almost everybody is now using AS 3.0+?

Comment (arturdryomov) [reply documentation/src/asciidoc/setting-up.adoc:157]: Not even close unfortunately (or otherwise). Especially noting that it is still a preview.

Comment (raniejade) [reply documentation/src/asciidoc/setting-up.adoc:157]: Okay, the only thing that is specific to AS 3.0+ is point `7` - I'll include it there.

File: documentation/src/asciidoc/setting-up.adoc (hunk 1 above)
Comment (artem-zinnatullin) [reply documentation/src/asciidoc/setting-up.adoc:168]: @ming13 well, rest of this `build.gradle` uses things like new API configurations `testImplementation` that simply won't work on AGP 2.x, so I'd maintain 3.x in the docs

Comment (arturdryomov) [reply documentation/src/asciidoc/setting-up.adoc:168]: @artem-zinnatullin That’s quite a mnemonic-like logic. I would make it explicit.

Comment (artem-zinnatullin) [reply documentation/src/asciidoc/setting-up.adoc:168]: @ming13 ? So we should add AGP 3.x here or I didn't understand you?

**Comment by raniejade** [comment]
Closing, no intention to merge this anymore.

**Pull request closed without merging**